catalog/spatial/registry/registry-schema-bindings/src/test/java/org/codice/ddf/registry/schemabindings/helper/InternationalStringTypeHelperTest.java

```diff
@@ -51,6 +64,13 @@ public void testGetString() throws Exception {
         InternationalStringType ist = getTestInternationalStringType();
 
         String istString = istHelper.getString(ist);
+
+        if (!isDefaultLocaleInTestSet()) {
```

Comment:
To avoid this conditional, could we use the `getTestInternationalStringType()` only in this test? We could have a separate test for the default Locale that populates the `InternationalStringType` with an entry for the default Locale and some other undefined Locale and makes sure the string for the default Locale is returned.

I believe this would allow us to remove `isDefaultLocaleInTestSet()` and all the logic related to that check.